### DISCUSSION TITLE
Hybrid Sequential Optimization with different variable blocks

### DISCUSSION THREAD

**Discussion by Vejahatf** [snl-dakota/dakota#63] [Q&A]
I was just wondering if Dakota has the capability of running a hybrid sequential optimization. where Variables blocks are different and best parameters from first Model's variable block are passed to second Model's variable block as state variable. for instance imagine the following case:

method
	id_method = 'HS'
	hybrid
		sequential
			method_pointer_list = 'GA1' 'GA2'

method
	id_method= 'GA1'	
	coliny_ea 
             model_pointer = 'M1' ....		
		
method
	id_method= 'GA2'
	coliny_ea 
            model_pointer = 'M2' .......
	
model
	id_model = 'M1'
	single
	    interface_pointer = 'I1'
	    variables_pointer = 'V1'
	    responses_pointer = 'R1'

model
	id_model = 'M2'
	single
	    interface_pointer = 'I1'
	    variables_pointer = 'V2'
	   responses_pointer = 'R1'

variables
	id_variables='V1'
  	continuous_design 7
             descriptors             'A'             'B'       'C'
             upper_bounds       0.04          50       35 
             initial_point       0.0001	    20     3.45 
            lower_bounds   0.00004	     1       0 

variables
	id_variables='V2'
  	continuous_design 2
              descriptors             'D'     'E'  
              upper_bounds        35      35   
              initial_point              0       0   
             lower_bounds        -35    -35    
        
    discrete_state_set
		real = 3
			descriptors = 'A' 'B'     'C'
			elements  are passed from model M1 
effectively I want second model to use best parameters from the first model. is this even possible or I have to do this manually as two optimization studies?

**Comment by jadamstephens** [comment]
Hi,

That seems like a neat feature, but, no.  Dakota does not seem to support it. When I tried to test it out a similar input, I got:

> Error: inconsistent counts in Variables::active_variables().

The error suggests to me that Dakota isn't paying attention to variable labels when it is mapping final solutions from the first optimization to initial points of the second. It just expects to find the same number of design variables in both variable blocks and maps them by order.
